examples/README.md

```diff
@@ -1,4 +1,5 @@
 # Examples
 
+* [Single project PR comment](single_project) - This example shows how to use `infracost comment` with Atlantis to post a comment to a comment to an open pr with a Infracost result.
```

Comment:
This example shows how to use `infracost comment` with Atlantis to post a cost estimate comment on new pull requests. 

(does the comment update when things change? if so, make that clear)